Modules/Common/src/ReferenceComparatorCheck.cxx

```diff
@@ -60,9 +62,14 @@ void ReferenceComparatorCheck::startOfActivity(const Activity& activity)
     mComparator->setThreshold(threshold);
   }
 
-  mActivity = activity;
```

Comment:
if `mActivity` is not used nor set I would simply remove it from this class

Reply:
@Barthelemy It is actually already removed, see https://github.com/AliceMCH/QualityControl/blob/4fd830b5a5e870d612ff2c42f0def61037bc9861/Modules/Common/include/Common/ReferenceComparatorCheck.h#L58

Or did I miss something? Thanks!

Reply:
sorry, indeed, I was checking the wrong branch when I saw it. 
All good !